lib/ff_dpdk_if.c

```diff
@@ -573,7 +585,9 @@ init_port_start(void)
     uint16_t i;
 
     for (i = 0; i < nb_ports; i++) {
-        uint8_t port_id = ff_global_cfg.dpdk.port_cfgs[i].port_id;
+        uint16_t port_id = ff_global_cfg.dpdk.portid_list[i];
+        struct ff_port_cfg *pconf = &ff_global_cfg.dpdk.port_cfgs[port_id];
+        nb_procs = pconf->nb_lcores;
```

Comment:
rename `nb_procs` to `nb_queues`  and remove  the above `uint16_t nb_procs = ff_global_cfg.dpdk.nb_procs;`.

Reply:
somewhere, the `nb_queues` of every port should be stored to `lcore_conf`, because `ff_rss_check` needs this value.

Reply:
`nb_lcores` in `struct ff_port_cfg` seems enough to get number of queues.

Reply:
yes, i just want to rename the `nb_procs`.